hack/.deployUtils

```diff
@@ -479,4 +479,57 @@ deployPrometheus() {
   kubectl config use-context kind-${clusterName}
   echo "Deploying Prometheus in ${clusterName}"
   wait_for "${KUSTOMIZE_BIN} --load-restrictor LoadRestrictionsNone build ${PROMETHEUS_DIR} --enable-helm --helm-command ${HELM_BIN} | kubectl apply -f -" "${PROMETHEUS_DIR} cluster config apply" "1m" "5"
+}
+
+deployGatekeeper(){
+  clusterName=${1}
+  kubectl config use-context kind-${clusterName}
+  echo "Deploying Gatekeeper in kind-${clusterName}"
+  ${HELM_BIN} repo add gatekeeper https://open-policy-agent.github.io/gatekeeper/charts
+  ${HELM_BIN} install gatekeeper/gatekeeper --name-template=gatekeeper --namespace gatekeeper-system --create-namespace
+  echo "test"
+  kubectl --context kind-${clusterName} wait --namespace=gatekeeper-system --for=condition=available --timeout=300s deployment/gatekeeper-audit
+  kubectl --context kind-${clusterName} wait --namespace=gatekeeper-system --for=condition=available --timeout=300s deployment/gatekeeper-controller-manager
+  kubectl --context kind-${clusterName} apply -f - <<EOF
+apiVersion: monitoring.coreos.com/v1
+kind: PodMonitor
+metadata:
+  name: gatekeeper-monitor
+  namespace: gatekeeper-system
+spec:
+  podMetricsEndpoints:
+    - port: metrics
+  selector:
+    matchLabels:
+      gatekeeper.sh/system: 'yes'
+EOF
+echo "Gatekeeper Ready"
+}
+
+configureGatekeeper() {
+  kubectl --context kind-${clusterName} apply -f ./config/api-upstream/gatekeeper/constraintTemplates.yaml
+  kubectl --context kind-${clusterName} wait  --for=condition=established --timeout=300s crd/requirepolicytargetinggateway.constraints.gatekeeper.sh
+  kubectl --context kind-${clusterName} apply -f ./config/api-upstream/gatekeeper/constraints.yaml
+  kubectl --context kind-${clusterName} apply -f ./config/api-upstream/gatekeeper/mutations.yaml
```

Comment:
Would you mind using kustomize here, similar to the other functions, so that relative files are not used.
That will make it easier for calling the function from a quickstart script that's been pulled down in isolation (without the rest of the repo).
Putting the 3 files into a kustomization file should do the trick normally, but I see there's a 'wait'.
Perhaps splitting into 2 kustomization files or using kfilt (there's an example elsewhere for filtering on CRDs first, then omitting them)